app/views/reminder_mailer/reminder_email.text.slim

```diff
@@ -0,0 +1,14 @@
+- rn2 = "\r\n\r\n"
+- dashs = "-" * 80
+- t4 = "\t" * 4
+- t3 = "\t" * 3
```

Comment:
Not sure if tabs in mails are reliable, you can also use string formatting to align with spaces (e.g. `%-20s` to left-align to 20 characters)